controller/src/main/java/org/jboss/as/controller/services/path/PathInfoHandler.java

```diff
@@ -100,6 +100,20 @@
                         .setDefaultValue(new ModelNode(MeasurementUnit.BYTES.getName()))
                         .build();
 
+    private static DateTimeFormatter getDateFormat() {
+        if (DATE_FORMAT == null) {
+            DATE_FORMAT = new DateTimeFormatterBuilder().appendInstant().appendZoneId().toFormatter(Locale.ENGLISH);
```

Comment:
Not really relevant for this PR, but we should probably use something like `DateTimeFormatter.ISO_OFFSET_DATE_TIME` so it's a standard format that could be parsed.

Current format:
```
2017-09-18T22:48:24.367ZAmerica/Los_Angeles
```
`DateTimeFormatter.ISO_OFFSET_DATE_TIME`
```
2017-09-18T15:48:24.367-07:00
```

Reply:
I guess another option is `DateTimeFormatter.ISO_ZONED_DATE_TIME` which would be
```
2017-09-18T15:56:21.875-07:00[America/Los_Angeles]
```

Reply:
+1. I opened https://issues.jboss.org/browse/WFCORE-3304